aws-redshiftserverless-namespace/template.yml

```diff
@@ -12,12 +12,12 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       Handler: software.amazon.redshiftserverless.namespace.HandlerWrapper::handleRequest
-      Runtime: java8
```

Comment:
Is there any reason we are only updating the version for namespace resource? And not for the workgroup resource because workgroup resource is also using java8.

Reply:
Yes, will update in separate PR, this is a priority as we received a sev2 from uluru about this